### ISSUE TITLE
Bookmarklet broken

### ISSUE THREAD

**Issue by cdelahousse** [SteamedPears/Code-Review#158]
http://review.steamedpears.com/bookmarklet/bookmarklet.js is returning a 404.

**Comment by spratt** [comment]
You're right, because it should be using bookmarklet.min.js.  At least, that's what the build script builds.

Also, there's a typo in the build script that outputs it to bookmarket/ instead of bookmarklet/.  See: http://review.steamedpears.com/bookmarket/bookmarklet.min.js

**Comment by spratt** [comment]
Fixed the typo in the build script: http://review.steamedpears.com/bookmarklet/bookmarklet.min.js

**Comment by cdelahousse** [comment]
Shit.
Nor is /bookmarklet/style.css being served.

**Comment by spratt** [comment]
Yeah, that file doesn't even get built by the build script.

**Comment by cdelahousse** [comment]
Also, the cors module isn't working.

**Comment by spratt** [comment]
What makes you say that?

**Comment by cdelahousse** [comment]
I'm getting this HTTP header/message:

OPTIONS
(failed)
Origin http://www.lipsum.com is not allowed by Access-Control-Allow-Origin.

**Comment by spratt** [comment]
Cool.  Things to fix.

**Comment by cdelahousse** [comment]
I'm fixing the CSS now...

**Comment by cdelahousse** [comment]
@spratt says "Does it work if you build the bookmarklet in dev mode? I think the root cause of this issue was that dev mode builds bookmarklet.js and production builds bookmarklet.min.js"

All the production script does is uglify the bookmarklet.js to a file named bookmarklet.min.js code. The code still runs fine. The issue I'm getting is that the server is no longer sending an OPTIONS header.

**Comment by cdelahousse** [comment]
Hold on. I'll retest locally.

**Comment by spratt** [comment]
I appreciate everything you're saying here, but the context of that question was a line of code that specifically references the uglified version.

**Comment by cdelahousse** [comment]
Even testing locally on the unminified build of the bookmarklet against my local install, I still get the same OPTION error.

**Comment by spratt** [comment]
Look: that question is about a GET request for bookmarklet.min.js when you may have only built bookmarklet.js.  The CORS issue is separate.

**Comment by cdelahousse** [comment]
Look: that's clearer. Thanks.
